src/index/txospenderindex.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <chainparams.h>
+#include <common/args.h>
+#include <index/disktxpos.h>
+#include <index/txospenderindex.h>
+#include <logging.h>
+#include <node/blockstorage.h>
+#include <validation.h>
+
+constexpr uint8_t DB_TXOSPENDERINDEX{'s'};
```

Comment:
(Saw this was somewhat covered in https://github.com/bitcoin/bitcoin/pull/24539#discussion_r864081172). Maybe document here?
```suggestion
// "Namespace" within LevelDB, separate from the one used by BaseIndex.
constexpr uint8_t DB_TXOSPENDERINDEX{'s'};
```

Reply:
done in https://github.com/bitcoin/bitcoin/pull/24539/commits/20b259332ff1d2e6ca6b291ffeda00b51497bbed

Reply:
Better than my suggestion, thanks!